kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -136,6 +176,12 @@ class BIOSPECIMEN(PropertyMixin):
         class EVENT_AGE(QuantityMixin):
             pass
 
+        class VOLUME(QuantityMixin):
+            pass
+
+        class CONCENTRATION(QuantityMixin):
+            pass
+
```

Comment:
I think we determined that volume/concentration aren't the exact right concepts for specimens. Good luck finding the relevant comment stream on notion, though. I don't know where it is or when it was written.